packages/app/src/components/Visualization/StartScreen.js

```diff
@@ -22,7 +22,7 @@ const StartScreen = ({ error, classes }) => {
             setMostViewedVisualizations(result.visualization)
         }
         fetchData(engine)
-    }, [])
+    }, [engine])
```

Comment:
wouldn't this make the function rerun if `engine` changes?

Reply:
It would.  Engine should be constant, so it shouldn’t be a problem in practice unless someone’s doing something funky with the provider, but you can also add ‘// eslint-ignore-line react-hooks/exhaustive-deps’ to prevent cli-style from complaining

Reply:
`eslint-ignore-line react-hooks/exhaustive-deps` 👍

Reply:
Added the eslint comment.
Now it raises a warning but the commit is allowed.

Reply:
What's the warning?  I don't think there should be one...

Reply:
Also it doesn't look like this line has changed?